README.md

```
@@ -153,6 +153,20 @@ dot -Tpdf cfg.dot > cfg.pdf
 
 </details>
 
+## Running unit tests
+
+Build with `VERIFIER_ENABLE_TESTS=ON` to produce the `tests` binary. Run the
+suite sequentially using:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)